### ISSUE TITLE
"PROJECT not set" is printed in every test run

### ISSUE THREAD

**Issue by wojtek-t** [kubernetes/test-infra#3602]
In every test run (of all suites), I'm currently seeing this log:
```
W0720 08:28:43.478] starts with local mode
W0720 08:28:43.479] PROJECT not set in job, will use local project
```

I think this is result of broken PR by @fejta  moving the env var to --gcp-project.

@fejta @kubernetes/test-infra-maintainers

**Comment by krzyzacy** [comment]
/assign

ref https://github.com/kubernetes/kubernetes/issues/49185#issuecomment-316773331

**Comment by krzyzacy** [comment]
fixed via #3610
